src/platform_impl/ios/view.rs

```diff
@@ -134,6 +134,18 @@ unsafe fn get_view_class(root_view_class: &'static Class) -> &'static Class {
                     height: screen_frame.size.height as f64,
                 }
                 .to_physical(scale_factor.into());
+
+                // If the app is started in landscape, the view frame and window bounds can be mismatched.
+                // The view frame will be in portrait and the window bounds in landscape. So apply the
+                // window bounds to the view frame to make it consistent.
+                let view_frame: CGRect = msg_send![object, frame];
+                let window_bounds: CGRect = msg_send![window, bounds];
+                if view_frame.size.width != window_bounds.size.width
+                    || view_frame.size.height != window_bounds.size.height
```

Comment:
There shouldn't be a problem with deriving `PartialEq` for `CGSize`, since we define it ourselves in the `ffi` module.

Reply:
Added `PartialEq` and just compared size. To be honest I don't know if I need to compare the `origin` as well. I haven't actively coded iOS for quite some number of years now. At least in the case of `winit` I think it doesn't really matter, we just have everything at `(0, 0)` anyway right?

Reply:
It doesn't look like anything guarantees a zero origin (see Discussion section on [`bounds`](https://developer.apple.com/documentation/uikit/uiview/1622580-bounds?language=objc) & [`frame`](https://developer.apple.com/documentation/uikit/uiview/1622621-frame?language=objc)), so we should probably just derive `PartialEq` for `CGRect` and compare the rects.

(I don't know off the top of my head if our particular usage leads to the origin always being zero, but that could end up changing anyway, so comparing the rects is the safest bet.)

Reply:
I've added `PartialEq` to `CGPoint` and `CGRect` and now compare the rects.